uktrade/data-hub-frontend
# Refactor/event details page

## Description of change

This PR refactors the `/events/{id}/details` and `/events/{id}/attendees` paths to React. It also fixes:

* Missing trailing segments in breadcrumbs
* The title in the header is now always the `{event-name}` (previously it was "Events" when on the `/events/{id}/details`} page

## Test instructions

1. Go to `/events`
2. Click on one of the events
3. You should land on `/events/{id}/details`
4. Click on the _Attendees_ item on the left hand side navigation
5. You should see
   * The _Attendees_ side bar navigation item highlighted as selected
   * Path in the address bar to have changed to `/events/{id}/attendees`, without page reloading
   * A list of attendees to the right of the sidebar
   * The breadcrumbs should be _Home > Events > {event-title} > Attendees_
6. Click on the _Details_ nav item
7. You should see
    * The _Details_ side bar navigation item highlighted as selected
   * Path in the address bar to have changed to `/events/{id}/details`, without page reloading
   * The event details table to the right of the sidebar
   * The breadcrumbs should be _Home > Events > {event-name} > Details_

## Screenshots

### Before

<img width="1022" alt="Screenshot 2024-12-06 at 10 49 09" src="https://github.com/user-attachments/assets/e7f13c9f-14f7-41ad-a0ee-00e039e684d0">

### After

<img width="1023" alt="Screenshot 2024-12-06 at 10 48 38" src="https://github.com/user-attachments/assets/81d110f9-a5c8-4a8f-bdfe-c00b307cbc9f">
